src/test/java/org/broadinstitute/hellbender/tools/walkers/validation/basicshortmutpileup/AllelePileupCounterUnitTest.java

```diff
@@ -0,0 +1,74 @@
+package org.broadinstitute.hellbender.tools.walkers.validation.basicshortmutpileup;
+
+import htsjdk.variant.variantcontext.Allele;
+import org.broadinstitute.hellbender.engine.ReadsDataSource;
+import org.broadinstitute.hellbender.utils.GATKProtectedVariantContextUtils;
+import org.broadinstitute.hellbender.utils.SimpleInterval;
+import org.broadinstitute.hellbender.utils.io.IOUtils;
+import org.broadinstitute.hellbender.utils.pileup.ReadPileup;
+import org.broadinstitute.hellbender.utils.test.BaseTest;
+import org.testng.Assert;
+import org.testng.annotations.DataProvider;
+import org.testng.annotations.Test;
+
+import java.nio.file.Path;
+import java.util.Collections;
+import java.util.List;
+
+public class AllelePileupCounterUnitTest extends BaseTest {
```

Comment:
After https://github.com/broadinstitute/gatk/pull/3475, this should extend `GATKBaseTest`

Reply:
Done